scss/_carousel.scss

```diff
@@ -195,6 +194,60 @@
   }
 }
 
+// Vertical slide transition
+// Writing below to override default indicators and control css
+.carousel-vertical {
+  .carousel-item-next:not(.carousel-item-start),
+  .active.carousel-item-end {
+    transform: translate(0, 100%);
+  }
+
+  .carousel-item-prev:not(.carousel-item-end),
+  .active.carousel-item-start {
+    transform: translate(0, -100%);
+  }
+
+  .carousel-indicators {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: unset;
+    flex-direction: column;
+    margin-right: 0;
+    margin-bottom: 0;
+
+    [data-bs-target] {
+      width: 15px;
+      height: 15px;
+      // stylelint-disable-next-line property-disallowed-list
+      border-radius: 50%;
```

Comment:
Any reason why you don't use our `border-radius` mixin?

Reply:
What is mixin? Never heard of it.
First I wrote border-radius: 50% but when I run npm run test it give me error that border-radius is not allowed so I saw in the spinner.md that they have included this CSS comment above the line and I did the same amd it worked so I keep it this way.

Reply:
Yeah, it's not allowed for a reason :) We have internal mixins we use, see the scss/mixins folder.

Reply:
I have now updated the file to use scss mixin. Hope this is what you wanted to do. Thank You.

Reply:
I don't think it's right, though. It should be `50%`, now it's 50rem. Maybe it's a limitation of the mixin. If so, you might need to revert to your previous solution.

Reply:
But I don't think that is gonna pose any problem because if we provide a radius of 50rem so it will eventually made that indicator in a circle which is our ultimate goal so reverting to previous version don't make difference.

Reply:
What you have right now doesn't work

![image](https://user-images.githubusercontent.com/349621/107373934-b9b05080-6aef-11eb-9b85-ca690876b220.png)

Reply:
Can you tell me the git command to revert to the previous commit i.e. commit `6f2d5c5` to undo these mixin changes. Thank you